boards/arm/nrf5340dk_nrf5340/doc/index.rst

```diff
@@ -43,32 +43,33 @@ nRF5340 SoC provides support for the following devices:
 .. figure:: img/nrf5340pdk.png
      :width: 711px
      :align: center
-     :alt: nRF5340 PDK
+     :alt: nRF5340 DK
```

Comment:
The image itself should also be updated.

Reply:
updated :)